cmd/options.go

```diff
@@ -95,6 +98,18 @@ func GetOptions(fs *flag.FlagSet) *Options {
 			nodeOptions.AddFlags(fs)
 			args = os.Args[1:]
 
+		case cmd == "pre-stop-hook":
+			clientset, clientErr := cloud.DefaultKubernetesAPIClient()
+			if clientErr != nil {
+				klog.ErrorS(err, "unable to communicate with k8s API")
+			} else {
+				err = hooks.PreStop(clientset, 1*time.Minute)
```

Comment:
Firstly, can we move the special time value to a constant at the top of this file called something like `preStopTimeout` so it isn't just a magic value?

Secondly, this value should be 30 seconds, as the default `terminationGracePeriod` (which we are using) is 30 seconds: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination

Reply:
Done.